commandapi-documentation-code/src/main/kotlin/dev/jorel/commandapi/examples/kotlin/ExamplesKotlinDSL.kt

```diff
@@ -303,7 +303,7 @@ commandAPICommand("remove") {
 }
 /* ANCHOR_END: argumentEntities1 */
 
-/* ANCHOR: argumentEntities2 */
+/* ANCHOR: argumentEntities4 */
```

Comment:
There aren't actually four Kotlin DSL examples, please undo this change.

Reply:
Revered 🎉